release.sh

```diff
@@ -1,13 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
 # Fail on errors and undefined variables
 set -eu
 
 # Be somewhere predictable
-cd "`dirname $0`"
+cd "$(dirname "$0")"
```

Comment:
I don't know how the double quotes behave here.

Reply:
This is quite common in shell scripts, I think it works as intended.